src/mscorlib/src/Internal/Runtime/Augments/AppContextAugments.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection;
+
+namespace Internal.Runtime.Augments
+{
+    /// <summary>For internal use only.  Exposes runtime functionality to the Appdomain implementation in corefx.</summary>
+    public static class AppContextAugments
```

Comment:
AppContext is part of System.Private.CoreLib (https://github.com/rahku/coreclr/blob/8070b5cdc41beaaa4f0bbf241029d7fb89906b60/src/mscorlib/model.xml#L150) so I was expecting to see you add it to that type.